howto/get-secrets/README.md

```diff
@@ -35,7 +35,7 @@ import (
 )
 
 func main() {
-  url := "http://localhost:3500/v1.0/secrets/kubernetes/my-secret"
+  url := "http://localhost:3500/v1.0/secrets/kubernetes/my-secret?metadata.namespace=default"
```

Comment:
Is the right side of this the namespace where the secret resource itself is?

```
metadata.namespace=default
```

It may clarify to put a comment above this line saying that.

Reply:
fixed